db/schema.rb

```diff
@@ -38,5 +38,15 @@
 
   add_index "products", ["category_id"], name: "index_products_on_category_id", using: :btree
 
+  create_table "users", force: :cascade do |t|
```

Comment:
Don't we need a username in the users table? The user story talks about a username. Do we want to use email when logging in instead?